### PR TITLE
Adding DaP and 18F analytics to the faq

### PR DESCRIPTION
**Pull request by geramirez**
Starting documentation for https://github.com/18F/analytics-standards/issues/17
feel free to make changes and edits.

### CODE REVIEW

**Comment by gbinal** [comment]
I offered a suggestion to modify the DAP language.

File: faq.md [hunk 1]
```diff
@@ -1,15 +1,24 @@
-### What is the difference between 18F Google Analytics and the Digital Analytics Program?  
-...
+### What is the difference between 18F Google Analytics and the Digital Analytics Program (DAP)?
 
-### Can I give access to a project's analytics to a client?  
+The [Digital Analytics Program](http://www.digitalgov.gov/services/dap/) and 18F Google Analytics are different services with two distinct code snippets.
```
Comment (jackiekazil) [inline faq.md]: @ramirezg  -- this is better, but still needs more clarification. Would folks participate in both or just one? Could folks participate in both?

Comment (geramirez) [inline faq.md]: Do we want to use the words "required" or "highly encouraged"?

Comment (jackiekazil) [inline faq.md]: Who is doing the encouraging or requiring it?

**Comment by gbinal** [comment]
After resolving #24, I'm in favor of merging this.

**Comment by gbinal** [comment]
Great - I'm going to go ahead and merge this.  We can edit further as we need.  

Good PR, @ramirezg.

**Pull request merged**